app/inputs/multi_value_input.rb

```diff
@@ -0,0 +1,64 @@
+class MultiValueInput < SimpleForm::Inputs::Base
```

Comment:
Another note about this class - it's technically opening up the MultiValueInput from hydra-editor. We should probably use a different class and adjust the field generator to use it (decorate the input to use our :as, maybe?)